trace/trace.go

```diff
@@ -278,7 +278,13 @@ func (s *Span) End() {
 		if s.spanContext.IsSampled() {
 			// TODO: consider holding exportersMu for less time.
 			exportersMu.Lock()
-			for e := range exporters {
+			for e, defaultAttrs := range exporters {
+				if len(defaultAttrs) > 0 && sd.Attributes == nil {
+					sd.Attributes = make(map[string]interface{})
+				}
+				for _, attr := range defaultAttrs {
+					sd.Attributes[attr.key] = attr.value
```

Comment:
Looks like the same SpanData is re-used when exporting to multiple exporters with the result that the second exporter gets all the attributes from the first, etc. 

I think we should either only support a single set of attributes, or else we need to clone the SpanData for each exporter.

Reply:
True, missed that.
Tbh I don't see a real use case in having different attributes on different exporters.
If others don't have one as well, then I'd suggest that we decouple it from the exporter.

We could create a `DefaultAttributes(a ...Attributes)` func that sets the default attrs on the package and then use them in `makeSpanData()` when the `Attributes` map is created.
This would allow for a more expected behavior when attribute keys collide: Default attrs would then be overridden by span attrs instead of the other way round.

WDYT?